run_project_tests.py

```diff
@@ -579,7 +579,7 @@ def check_file(fname):
     with open(fname, 'rb') as f:
         lines = f.readlines()
     for line in lines:
-        if b'\t' in line:
+        if line.startswith(b'\t'):
```

Comment:
Without this, you can't even use `\t` inside a string for testing. I can change it to a regex, but I wasn't sure if it was worth the trouble; people who mix spaces and tabs usually use `<tab><tab><spaces>`; I've never seen `<spaces><tab>`.